.github/archive/test_failures.yml

```diff
@@ -0,0 +1,12 @@
+name: OnPurpose Failure
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+jobs:
+  failure_test:
+    if: contains(github.repository, 'dougch/s2n')
```

Comment:
Are we using `dougch/s2n` for testing purpose? Can we use `awslabs/s2n` instead?

Reply:
this was for private testing- we don't need (any more) false alarms on the public repo